### PR TITLE
Added buttons to user menu in telegram bot + fixed PEP8

### LINKED ISSUE #66: Implement user menu in telegram bot

### PR DESCRIPTION
**Pull request by kushnir-tadey**
closes #66

### CODE REVIEW

File: TelegramBot/handlers/user_menu.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+BUTTON_SHOW_MY_ORDERS = "Show my orders"
+BUTTON_PROFILE = "My profile"
+
+
+def get_base_reply_keyboard():
+    """
+    Register keyboard buttons into the menu
+    """
+    keyboard = [
+        [
```
Comment (v-lavrushko) [inline TelegramBot/handlers/user_menu.py:18]: that's good, but what would be better - if buttons show up if the order is made, address is created etc...

Comment (v-lavrushko) [reply TelegramBot/handlers/user_menu.py:18]: can do it as issue for the next sprint

**Review by OlehHnyp — APPROVED** [review]

**Review by OlehHnyp — APPROVED** [review]
python-telegram-bot нема в requirements.txt

**Review by strangerok1507 — APPROVED** [review]

**Review by MykhailoGrynenko — APPROVED** [review]
LGTM

**Pull request merged**